core/src/main/java/org/apache/accumulo/core/client/admin/FindMax.java

```diff
@@ -181,4 +180,9 @@ public static Text findMax(Scanner scanner, Text start, boolean is, Text end, bo
 
     return _findMax(scanner, start, is, end, ie);
   }
+
+  public static Text findMax(Scanner scanner, RowRange rowRange) {
```

Comment:
I think you just want this one `findMax` method that uses the RowRange parameter.